### PR TITLE
migrated website to docusaurus 2

### PR DESCRIPTION
**Pull request by ozgunbal**
- Live version: https://build.ozgunbal.now.sh/
- I only removed github star counts below `Get Started` button at v1:
image
It can be added if it's necessary but IMHO, doesn't look good with new theme.
- Hey, @yangshun and @endiliey . You can check it when you're available.
- This PR related to #46 and https://github.com/facebook/docusaurus/issues/1834

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
-    "rename-version": "docusaurus-rename-version"
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy"
   },
-  "devDependencies": {
-    "docusaurus": "^1.0.9"
+  "dependencies": {
+    "@docusaurus/core": "^2.0.0-alpha.26",
```
Comment (yangshun) [inline website/package.json]: We've just published .27, could you upgrade to that?

Comment (ozgunbal) [reply website/package.json]: done

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    navbar: {
+      title: "Profilo",
+      logo: {
+        alt: "Profilo Logo",
+        src: "img/profilo_logo_small.png"
+      },
+      links: [
+        { to: "docs/getting-started", label: "Docs", position: "left" },
+        {
+          href: "https://github.com/facebook/docusaurus",
```
Comment (yangshun) [inline website/docusaurus.config.js]: This should be updated

Comment (ozgunbal) [reply website/docusaurus.config.js]: done. live version is updated, too.

File: website/src/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      </header>
+      <main>
+        {features && features.length && (
+          <section className={styles.features}>
+            <div className="container">
+              <div className="row">
+                {features.map(({ title, description }, idx) => (
+                  <div
+                    key={idx}
+                    className={classnames("col col--6", styles.feature)}
```
Comment (lex111) [inline website/src/pages/index.js]: Class `feature` does not exist.

```suggestion
                    className="col col--6"
```

File: website/src/pages/index.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+          <p className="hero__subtitle">{siteConfig.tagline}</p>
+          <p className="lead">
+            Profilo is an Android library for collecting performance traces from
+            production builds of an app.
+          </p>
+          <div className={styles.buttons}>
+            <Link
+              className={classnames(
+                "button button--secondary button--lg",
+                styles.getStarted
```
Comment (lex111) [inline website/src/pages/index.js]: Should be removed.

File: website/src/pages/index.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+  return (
+    <Layout
+      title={`${siteConfig.title} - ${siteConfig.tagline}`}
+      description={siteConfig.tagline}
+    >
+      <header className={classnames("hero hero--primary", styles.heroBanner)}>
+        <div className="container">
+          <h1 className="hero__title">{siteConfig.title}</h1>
+          <p className="hero__subtitle">{siteConfig.tagline}</p>
```
Comment (lex111) [inline website/src/pages/index.js]:
```suggestion
          <p>{siteConfig.tagline}</p>
```

File: website/src/pages/index.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+          <p className="lead">
```
Comment (lex111) [inline website/src/pages/index.js]:
```suggestion
          <p>
```

File: website/src/pages/index.js (hunk 3 above)
Comment (ozgunbal) [reply website/src/pages/index.js]: done

File: website/src/pages/index.js (hunk 4 above)
Comment (ozgunbal) [reply website/src/pages/index.js]: done

File: website/src/pages/index.js (hunk 6 above)
Comment (ozgunbal) [reply website/src/pages/index.js]: done

**Comment by ricardorey10** [comment]
Hey, thanks so much for taking care of this migration.

One quick thing, though. Any chance we can make the font of all the code snippets a big bigger? It's kind of hard to read right now :(

**Comment by ricardorey10** [comment]
@ozgunbal 

I pulled the PR but just noticed that it explicitly says that it should not be landed. I'll wait for the final version.

**Comment by yangshun** [comment]
I agree regarding the font size. But maybe we could handle it at the Docusaurus level. This issue is not specific to Profilo. Thanks for reviewing @ricardorey10!

**Comment by ricardorey10** [comment]
@yangshun @ozgunbal 

Looks like I saw the diff while it was in some unfinished state, and I don't see anything about it not being ready to land yet, so I'll get that landed soon. Thanks for taking care of this!

**Comment by ricardorey10** [comment]
Hey, @yangshun @ozgunbal it looks like there are some errors related to your commit. Would you mind taking a look at that?

**Comment by ozgunbal** [comment]
hi, i assume font-size would be handled via docusaurus , right @yangshun ? @ricardorey10  other than that, I can't see details of failed `Facebook Internal - Linter` and `Facebook Internal - Builds & Test`. What should I update at current state of PR?

**Pull request closed without merging**